enterprise/course_catalog_api.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to get details from the course discovery API.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+try:
```

Comment:
We should try/except only the code we're concerned about, rather than all imports, since the except only deals with one use case.